server/tls.go

```diff
@@ -0,0 +1,81 @@
+package server
```

Comment:
Mh, no, not really. :D It is always up to the client how to import packages and at the end there will be some `ulogger` package and the client has to rename it anyway. Lets keep it as it is and the package names somehow reasonable. We import already like `microerror` and `micrologger`. `microtls` would follow this pattern quite nicely IMO.